docs/source/en/training/text2image.mdx

```diff
@@ -104,25 +110,19 @@ accelerate launch train_text_to_image.py \
   --lr_scheduler="constant" --lr_warmup_steps=0 \
   --output_dir=${OUTPUT_DIR}
 ```
```

Comment:
Can we also show here how to do checkpointing and intermediate logging and uploading to the Hub?